css-two-column-layout/index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Challenge - Two Column Layout</title>
+  <link rel="stylesheet" href="fonts.css">
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <div class="container">
+    <div class="row">
+      <div class="column-full item">
+        <h1>SCALE, SPEED, AGILITY</h1>
+        <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Quaerat molestiae modi tenetur aperiam iusto dolore autem pariatur. Quibusdam, omnis ad!</p>
+      </div>
+    </div>
+    <div class="row">
+       <div class="column-one-half row feature">
+          <div class="column-one-fourth">
+            <img src="images/software.svg">
+          </div>
+          <div class="column-three-fourth">
+            <h2>intuitive,<br />smart software</h2>
+            <p>Lorem ipsum dolor sit, amet consectetur adipisicing elit. Reiciendis, labore quis voluptatibus. Sunt recusandae voluptates dolores deleniti repellendus reprehenderit incidunt ab vel?</p>
+          </div>
+        </div>
+         <div class="column-one-half row feature">
+            <div class="column-one-fourth">
+              <img src="images/practice.svg">
+            </div>
+            <div class="column-three-fourth">
+              <h2>practice medicine, not administration</h2>
+              <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Impedit, asperiores temporibus alias repellat ab facilis cupiditate id porro blanditiis tenetur sunt saepe nemo voluptas nulla?</p>
+            </div>
+          </div>
+          <div class="column-one-half row feature">
+              <div class="column-one-fourth">
+                <img src="images/circumstance.svg">
+              </div>
+              <div class="column-three-fourth">
+                <h2>for any circumstance</h2>
+                <p>Lorem ipsum dolor sit amet consectetur, adipisicing elit. Consequatur quas magnam eveniet exercitationem fugit odit reprehenderit facere maiores iusto commodi.</p>
+              </div>
+            </div>
+            <div class="column-one-half row feature">
+                <div class="column-one-fourth">
+                  <img src="images/data.svg">
+                </div>
+                <div class="column-three-fourth">
+                  <h2>data that transforms</h2>
+                  <p>Lorem ipsum dolor, sit amet consectetur adipisicing elit. Sequi tempore aliquid repudiandae doloremque, error distinctio quia nulla ab odio. Veniam..</p>
+                </div>
+              </div>
+            </div>
+             <button>learn more</button>
+          </div>
```

Comment:
Oh dear, it looks like your indentation got a bit messed up here. Please revise this so that I can more easily read through it to give you feedback 🙏